sdk/synapse/synapse-artifacts/src/models/parameters.ts

```diff
@@ -544,7 +576,7 @@ export const filterParameters: OperationParameter = {
   mapper: RunFilterParametersMapper
 };
 
-export const runId: OperationURLParameter = {
+export const runId1: OperationURLParameter = {
```

Comment:
Is this duplicate schema ?

Reply:
Yes, this is introduced by runNotebook operation group.